packages/cardpay-sdk/sdk/safes/base.ts

```diff
@@ -128,6 +128,14 @@ const safesQuery = `
   }
 `;
 
+// TODO: replace this with a formal implementation
+export async function viewSafe(network: 'xdai' | 'sokol', safeAddress: string): Promise<Safe | undefined> {
+  let {
+    data: { safe },
+  } = await query(network, safeQuery, { id: safeAddress });
+  return processSafeResult(safe as GraphQLSafeResult);
+}
```

Comment:
@habdelra on this page, we need to display merchant customization details to a user that may not be connected to layer 2, using a merchant safe address and network extracted from the URL.

I'd like to open a PR to expose a way to view a safe without needing a web3 instance, would this be okay? Wondering if there is a better way, because this seems to break the SDK's conventions.

Reply:
I'm totally open to that

Reply:
Cool, will do that tomorrow then. Thanks!